css/index.css

```diff
@@ -104,14 +149,90 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+.one {
+    background-color: teal;
+    animation: pulse 10s infinite;  
+}
+
+.two {
+    background-color: gold;
+    animation: pulse 5s infinite;
+}
+
+.three {
+    background-color: cadetblue;
+    animation: pulse 3s infinite;
+}
+
+.four {
+    background-color: coral;
+    animation: pulse 20s infinite;
+}
+
+.five {
+    background-color: crimson;
+    animation: pulse 8s infinite;
+}
+
+.six {
+    background-color: forestgreen;
+    animation: pulse 14s infinite;
+}
+
+.seven {
+    background-color: darkorchid;
+    animation: pulse 33s infinite;
+}
+
+.eight {
+    background-color: hotpink;
+    animation: pulse 7s infinite;
+}
+
+.nine {
+    background-color: indigo;
+    animation: pulse 17s infinite;
+}
+
+.ten {
+    background-color: dodgerblue;
+    animation: pulse 22s infinite;
+}
+
```

Comment:
Glad to see you having colors here! I'm glad I trusted you when you showed me your project on Friday ;)